modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -53,9 +54,22 @@ public BrowserWebDriverContainer() {
 
     }
 
+    /**
+     * Constructor taking a specific webdriver container name and tag
+     * @param dockerImageName
+     */
+    public BrowserWebDriverContainer(String dockerImageName) {
+        super.setDockerImageName(dockerImageName);
+        this.imageNameIsSet = true;
```

Comment:
Unfortunately, no; `GenericContainer` sets a default image name (`alpine:3.2`) on the no-args constructor. I agree with the principle, though...